dirsrvtests/tests/suites/basic/basic_test.py

```diff
@@ -1713,6 +1713,38 @@ def test_dscreate_with_different_rdn(dscreate_test_rdn_value):
         else:
             assert True
 
+def test_conntablesize_attr_dse(topo):
```

Comment:
This test may be skipped on main and after 2.3

Reply:
The test runs on branches 1.4.3 -> 2.2